packages/tests/server/routerMeta.test.ts

```diff
@@ -77,3 +77,155 @@ test('route meta in middleware', async () => {
   expect(middleware).toHaveBeenCalledTimes(2);
   await close();
 });
+
+test('default meta', async () => {
+  const t = initTRPC
+    .meta<{
+      data: string;
+    }>({ data: 'foobar' })
+    .create();
```

Comment:
I'm thinking that this API might be nicer?:

```ts
  const t = initTRPC
    .meta<{
      data: string;
    }>()
    .create({ defaultMeta: { data: 'foobar' } });
```

Considering that we do all other "runtime stuff" at the `create()`?

Reply:
Okay that's fair enough! It might prevent people from inferring their meta via generics, but maybe there's not much value (or it's worth actively discouraging) that?

Right now something like this should be possible:
```ts
const t = initTRPC
	  .meta({
	    data: "foobar"
	  })
          // ^ infer { data: string }
	  .create();
```

Reply:
Changes pushed!